cachito/web/models.py

```diff
@@ -203,13 +203,13 @@ def to_purl(self):
             purl_name = urllib.parse.quote(self.name, safe="")
             return f"pkg:golang/{purl_name}@{self.version}"
 
-        elif self.type == "npm":
+        elif self.type == "npm" or self.type == "yarn":
             purl_name = urllib.parse.quote(self.name)
             match = re.match(
                 r"(?P<protocol>[^:]+):(?P<has_authority>//)?(?P<suffix>.+)", self.version
             )
             if not match:
-                return f"pkg:npm/{purl_name}@{self.version}"
+                return f"pkg:{self.type}/{purl_name}@{self.version}"
```

Comment:
@athos-ribeiro may correct me on this, but I think we want to use `pkg:npm` for both NPM and Yarn

Reply:
changed it back to npm

Reply:
For context, this is the doc that we reference when deciding what a purl should look like: https://github.com/package-url/purl-spec
You will see that npm is defined there but yarn is not, so we're going with npm here.